src/Workspaces/MSBuild/Core/MSBuild/MSBuildProjectLoader.cs

```diff
@@ -172,44 +172,26 @@ public async Task<SolutionInfo> LoadSolutionInfoAsync(
                 // TryGetAbsoluteSolutionPath should throw before we get here.
                 return null!;
             }
-
+            var isSolutionFilter = SolutionFilterReader.IsSolutionFilterFilename(absoluteSolutionPath);
```

Comment:
Nit: Newline after closing brace
```suggestion

            var isSolutionFilter = SolutionFilterReader.IsSolutionFilterFilename(absoluteSolutionPath);
```